tsk/auto/db_postgresql.cpp

```diff
@@ -749,20 +749,15 @@ int TskDbPostgreSQL::createIndexes() {
         attempt_exec("CREATE INDEX relationships_data_source_obj_id  ON account_relationships(data_source_obj_id);",
             "Error creating relationships_data_source_obj_id index on account_relationships: %s\n") ||
         //tsk_events indices
-        attempt_exec("CREATE INDEX events_data_source_obj_id  ON tsk_events(data_source_obj_id);",
-            "Error creating relationships_data_source_obj_id index on tsk_events: %s\n") ||
-        attempt_exec("CREATE INDEX events_event_id_hash_hit  ON tsk_events(event_id, hash_hit);",
-            "Error creating events_event_id_hash_hit index on tsk_events: %s\n") ||
-        attempt_exec("CREATE INDEX events_event_id_tagged  ON tsk_events(event_id, tagged);",
-            "Error creating events_event_id_tagged index on tsk_events: %s\n") ||
-        attempt_exec("CREATE INDEX events_file_obj_id  ON tsk_events(file_obj_id);",
-            "Error creating events_file_obj_id index on tsk_events: %s\n") ||
-        attempt_exec("CREATE INDEX events_artifact_id  ON tsk_events(artifact_id);",
-            "Error creating events_artifact_id index on tsk_events: %s\n") ||
-        attempt_exec("CREATE INDEX events_sub_type_time  ON tsk_events(sub_type,  time);",
+        attempt_exec("CREATE INDEX events_data_source_obj_id  ON tsk_event_descriptions(data_source_obj_id);",
```

Comment:
I removed some of the indexes that seemed less useful or didn't translate directly to the new schema.